client-blog-post.md

```diff
@@ -0,0 +1,128 @@
+# Official Memgraph Drivers Are Out!
+
+We're happy to announce that Memgraph drivers for Python and C languages, as
+well as Memgraph CLI are now published. You can find their GitHub repositories
+here:
+
+- [mgclient] (C driver)
+- [pymgclient] (DB-API 2.0 compliant Python driver)
+- [mgconsole] (command line interface)
+
+## How to get started
+
```

Comment:
Maybe add a sentence about what's to come next. Something like(e.i. We will now install mgclient, pymgclient and mgconsole and run a few basic queries)